src/do_command.c

```diff
@@ -589,6 +589,12 @@ static int child_process(entry * e, char **jobenv) {
 				Debug(DPROC, (", dumped core"));
 			Debug(DPROC, ("\n"));
 	}
+	if ((e->flags & DONT_LOG) == 0) {
+		char *x = mkprints((u_char *) e->cmd, strlen(e->cmd));
+
+		log_it(usernm, getpid(), "CMDEND", x, 0);
```

Comment:
You need to check for x being NULL here. I.E. replace x with `x ? x : "*unknown*"` or something.

Reply:
I can do, but when printing out the command initially (with the CMD log entry), it makes no such check (around line 197 in do_command.c)

Reply:
Then that place should be fixed as well, but that should be a separate PR or I'll fix it myself.